### PR TITLE
Define the eBPF network sampling strategy

### PR DESCRIPTION
**Pull request by mrproliu**
Now we have the network profiling based on eBPF technology, then we can capture all the request and response data.
Therefore, we can add some sampling policies to match the time we can upload them to the backend.

### CODE REVIEW

**Comment by wu-sheng** [comment]
I did some polish to the protocols. But I didn't test it locally. Please review and recheck.
I am making the settings easier to understand.

File: ebpf-profiling.graphqls [hunk 1]
```diff
@@ -30,10 +30,45 @@ input EBPFProfilingTaskFixedTimeCreationRequest {
...
+    # matches all requests if not set
+    uriRegex: String
+
+    # the minimal request duration to activate the network data(HTTP request/response raw data) sampling.
+    # Collecting requests without minimal request duration
+    minDuration: Int
+    # Collecting requests when the response code is 400-499
+    when4xx: Boolean!
+    # Collecting requests when the response code is 500-599
+    when5xx: Boolean!
```
Comment (wu-sheng) [inline ebpf-profiling.graphqls:58]: `minDuration` condition could work when `when5xx` or `when4xx` matched. If neither of them is set, `minDuration` works on its own.

Comment (mrproliu) [reply ebpf-profiling.graphqls:58]: If the `when5xx` is matched, but the duration of the response is small than `minDuration`, should be sampling?

Comment (wu-sheng) [reply ebpf-profiling.graphqls:58]: No, we should not.

Comment (mrproliu) [reply ebpf-profiling.graphqls:58]: Sure, let me verify the format. Once the format verifies success, I think it could be merged.

**Pull request merged**